src/overlays/misc/ovl_kaleido_scope/z_kaleido_collect.c

```diff
@@ -511,7 +511,11 @@ void KaleidoScope_DrawQuestStatus(PlayState* play, GraphicsContext* gfxCtx) {
                     gDPPipeSync(POLY_OPA_DISP++);
 
                     if (D_8082A124[sp218] == 0) {
+#if PLATFORM_N64
+                        gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 80, 150, 255, D_8082A150[sp218]);
+#else
                         gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 80, 255, 150, D_8082A150[sp218]);
+#endif
```

Comment:
make a macro for these RGB values, similar to A_BUTTON_R? Seems to show up several times